conda-forge.yml

```
@@ -7,3 +7,6 @@ github:
 provider:
   linux_aarch64: default
   linux_ppc64le: default
+noarch_platforms:
+  - linux_64
+  - win_64
```

Swapping to keep keys sorted. Otherwise no-op bot migrations that change nothing besides these keys would be opened. Hopefully this avoids them

```suggestion
noarch_platforms:
  - linux_64
  - win_64
provider:
  linux_aarch64: default
  linux_ppc64le: default
```